delphi/translators/GrFN2WiringDiagram/SIR-simple__functions.jl

```diff
@@ -0,0 +1,19 @@
+function SIR_simple__sir__assign__I__0(I, infected, recovered)
```

Comment:
Is this superseded by the SIR-simple__functions--correct.jl file? If so we should remove the incorrect version.